src/QueueReceiver.Core/Interfaces/IPersonRepository.cs

```diff
@@ -1,13 +1,16 @@
 ﻿using QueueReceiver.Core.Models;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace QueueReceiver.Core.Interfaces
 {
     public interface IPersonRepository
     {
-        Task<Person> FindByUserOid(string userOid);
-        Task<Person> FindByUserEmail(string userEmail);
-        Task<Person> FindByUsername(string userName);
+        Task<Person?> FindByUserOid(string userOid);
+        //Task<Person> FindByUserEmail(string userEmail);
```

Comment:
remove? also next line

Reply:
fixed in different branch, leaving it to avoid too much merge conflicts